development-certs/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM busybox
+COPY . /
```

Comment:
"Dependent sources" are basically directories whose source code is shared between the build process of multiple images, such as the case of loading development certs into the images by default.

It used to be that I did some weird inception work to fit these source directories together and have Dockerfiles find them during the build process that way, but I discovered a simpler approach:

If you have a directory whose source code you wish to share with multiple docker images, just build a docker image of that source code itself (as I do here) and then copy from one image to the new image using the multi stage build process.  A lot simpler than it sounds, example below.